app/src/main/java/com/lagradost/cloudstream3/metaproviders/TmdbProvider.kt

```diff
@@ -264,6 +268,51 @@ open class TmdbProvider : MainAPI() {
         return null
     }
 
+    open suspend fun fetchContentRating(id: Int?, country: String, isTvSeries: Boolean): String? {
+        id ?: return null
+        return if (isTvSeries) {
+            val contentRatings = tmdb.tvService().content_ratings(id).awaitResponse().body()
+            contentRatings?.results
+                ?.find { it.iso_3166_1 == country }
+                ?.rating
+        } else {
+            val releaseDates = tmdb.moviesService().releaseDates(id).awaitResponse().body()?.results
+            val certification = findFirstNonEmptyCertificationWithCountry(releaseDates, country)
+            if (!certification.isNullOrEmpty()) {
+                certification
+            } else {
+                null
+            }
+        }
+    }
+
+    private fun findFirstNonEmptyCertificationWithCountry(
+        releaseDatesResults: List<ReleaseDatesResult>?,
+        country: String
+    ): String? {
+        for (releaseDateResult in releaseDatesResults.orEmpty()) {
+            if (releaseDateResult.iso_3166_1 == country) {
+                val certification = findFirstNonEmptyCertification(releaseDateResult.release_dates)
+                if (!certification.isNullOrEmpty()) {
+                    return certification
+                }
+            }
+        }
+
+        return null
+    }
+
+    private fun findFirstNonEmptyCertification(releaseDates: List<ReleaseDate>?): String? {
```

Comment:
Take advantage of kotlin functions :pray: 

`list.firstOrNull { !it.isNullOrBlank() }`

(.find is the same thing)